react-app/src/components/ActionForm/actionForm.components.tsx

```diff
@@ -1,7 +1,7 @@
 import { Link } from "react-router";
 import { z } from "zod";
 
-import { FAQ_TAB } from "@/router";
+import { FAQ_TAB } from "@/utils";
```

Comment:
This is fine but we should really add an @/constants instead, that way its like "i need a constant" not a utility

Reply:
Fixed with https://github.com/Enterprise-CMCS/macpro-mako/pull/1301/commits/0edbb9df4b361856ee69df7b9e2e99d9fbd4ebdb